cg/store/models.py

```diff
@@ -1056,26 +1056,28 @@ class SampleRunMetrics(Base):
     __tablename__ = "sample_run_metrics"
     id: Mapped[PrimaryKeyInt]
     sample_id: Mapped[int] = mapped_column(ForeignKey("sample.id"))
-    run_metrics_id: Mapped[int] = mapped_column(ForeignKey("run_metrics.id"))
+    instrument_run_id: Mapped[int] = mapped_column(ForeignKey("instrument_run.id"))
     type: Mapped[DeviceType]
 
-    run_metrics: Mapped[RunMetrics] = orm.relationship(back_populates="sample_metrics")
+    instrument_run: Mapped[InstrumentRun] = orm.relationship(back_populates="sample_metrics")
     sample: Mapped[Sample] = orm.relationship(back_populates="_new_run_metrics")
 
     __mapper_args__ = {
         "polymorphic_on": "type",
     }
 
 
-class IlluminaSampleRunMetrics(SampleRunMetrics):
+class IlluminaSampleSequencingMetrics(SampleRunMetrics):
```

Comment:
I also have in my notes `IlluminaSequencingRunMetrics`, but maybe I got that wrong

Reply:
I think `IlluminaSampleSequencingMetrics` is what we settled on